src/bundle/Command/CronRunCommand.php

```diff
@@ -46,6 +47,30 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $cron->setExecutor(new Executor());
         $cron->setResolver($resolver);
 
-        $cron->run();
+        $reports = $cron->run();
+
+        while ($cron->isRunning()) {}
+
+        if($this->getContainer()->has('monolog.logger.cronjob')){
```

Comment:
Use dependency injection for logger (make the command implement `LoggerAwareInterface` and tag the service with `{ name: monolog.logger, channel: cronjob }`)

Reply:
@Nattfarinn How do I have to implement `LoggerAwareInterface `? In the latest commit I inject `LoggerInterface` like @alongosz  has recommended.

Reply:
@homarx `LoggerInterface $cronjobLogger` is fine. I wasn't aware of @alongosz answer at the time of writing.